e2e-polybft/cardanofw/test-apex-chain.go

```diff
@@ -15,7 +15,7 @@ type ITestApexChain interface {
 	CreateWallets(validator *TestApexValidator) error
 	CreateAddresses(bladeAdmin *crypto.ECDSAKey, bridgeURL string) error
 	FundWallets(ctx context.Context) error
-	RegisterChain(validator *TestApexValidator) error
+	RegisterChain(validator *TestApexValidator, system string) error
```

Comment:
`system string` should be some config parameter `IsSkyline`  or `IsReactor`. interface method should not be changed